src/server/components/lists/controllers/index.ts

```diff
@@ -70,15 +75,22 @@ export async function listsPostController(
     }
   }
 
+  const queryString = queryStringFromParams(params);
   res.redirect(`${listsRoutes.finder}?${queryString}`);
 }
 
 export function listsGetController(req: Request, res: Response): void {
-  const params = getAllRequestParams(req);
+  let params = getAllRequestParams(req);
+
   if (params.page === undefined || params.page !== "") {
     params.page = "";
   }
   const queryString = queryStringFromParams(params);
+  if (params.country) {
+    const countryName: string = formatCountryParam(params.country as string);
```

Comment:
Really small comment but how comes the country variable from like **41** was not used here?"

Reply:
so the country variable is retrieved from the URL parameters and we can't guarantee the case of this value.  The call to formatCountryParam ensures the case is always correct prior to querying the database so it's matching like for like.